doc/source/setup-jupyterhub.rst

```diff
@@ -115,6 +115,15 @@ Install JupyterHub
       ``helm delete --purge <YOUR-RELEASE-NAME>``. Then reinstall by repeating this
       step. If it persists, also do ``kubectl delete <YOUR-NAMESPACE>`` and try again.
 
+    .. note::
+
+      If you get the error ``cannot get namespaces in the namespace
+      "<your-namespace>"``, it could be because of a permissions
+      issue. Run `this command
+      <https://kubernetes.io/docs/admin/authorization/rbac/#permissive-rbac-permissions>`_
+      to give all users permissions to configure your cluster,
+      though be warned that it may cause insecurities in your cluster.
```

Comment:
it will make your cluster insecure, so let's make sure we mention that very clearly.

Also mention that there should be a fix for this soon.